insight-rest-objects/src/main/java/org/nuxeo/ai/sdk/objects/Bucket.java

```diff
@@ -24,7 +24,14 @@
 import org.apache.commons.lang3.StringUtils;
 
 /**
- * POJO for defining statistics' buckets
+ * POJO for defining statistics' bucket retrieved from ES
+ *
+ * <code>
+ * {
+ *  "key":"dc:title",
+ *  "docCount":100
+ * }
+ * </code>
  */
 public class Bucket {
```

Comment:
I really do not like the name Bucket. This is a pure name from ES aggregation return, and nothing to do with statistics (counts in here=
The name should be more related to the content, so something like KeyCount, counter. 

In reality, and since we are doing this. We should just have Key : name, value: count.
The current format is fully attached to the outputs from ES, and that is not a good idea.

Reply:
all the name changes are not part of the ticket. Other projects depend on it. If you have suggestions, please create a ticket for the next version, so we can accommodate those changes.